### ISSUE TITLE
OCR: Flawed hardcoded rule for converting to uppercase

### ISSUE THREAD

**Issue by karussellbremser** [SubtitleEdit/subtitleedit#5340]
I sometimes have problems during OCR concerning auto-converting initial characters to uppercase when the option "Tools->Fix common OCR errors - also use hard-coded rules" is active and obviously also "Fix OCR errors" is on during OCR. 

Here is an example for my problem from the attached sup file: 
l. 13:
<i>But the senseless killing
of Karen Marie White,</i>

l. 14:
<i>a young woman murdered
in the prime of her life,</i>

l. 15:
<i>Ieaves us all with a feeling
of anger and hopelessness.</i>

In l. 15, "Ieaves" is written with uppercase i, not with lowercase L. This is despite the fact that I added a custom OCR replace rule for this word from "Ieaves" to "leaves" with lowercase L. There must be some hardcoded rule which changes it back to the erroneous version despite my custom replacement rule.
What's worse is that the erroneous word does not even show up in the "Unknown words" section so that I could fix it manually.

Is it conceivable to either fix this specific problem or add an option to turn off only auto-capitalization of initial characters? (I don't want to turn off the hardcoded rules completely as they are generally helpful.)

Thanks and best regards

[Howling_II_track5_[eng].zip](https://github.com/SubtitleEdit/subtitleedit/files/7150469/Howling_II_track5_.eng.zip)

**Comment by niksedk** [comment]
Hm, works fine here:
![image](https://user-images.githubusercontent.com/1261331/133295344-0017cbb7-07a5-4c18-8f7f-f991e8ecd9e0.png)

What settings do you use?

**Comment by karussellbremser** [comment]
I tested with your settings and indeed get the same result as you. I found out that turning off "Try to guess unknown words" causes the difference, this option seems to "fix" the problem. But the root problem is still there. 

I also now realized that in the "All fixes" category it says:

`#15: <i>leaves us all with a feeling of anger and hopelessness.</i> ⇒ <i>Ieaves us all with a feeling of anger and hopelessness.</i>`
(lowercase L changed to uppercase i)

So there must be some erroneous internal rule to do that. It is not within my custom word replacements. And this internal rule silently overrides my existing custom replacement rule from `Ieaves` to `leaves`.
My guess is that it has something to do with being the first letter of the line.
